src/amber/validations/params.cr

```diff
@@ -55,7 +55,7 @@ module Amber::Validators
     getter raw_params = Amber::Router::Params.new
     getter errors = [] of Error
     getter rules = [] of BaseRule
-    getter params = {} of String => String?
+    getter params = {} of String => String? | Amber::Router::Files::File
```

Comment:
This union type is expressed several places, is it worth having an alias for it?

Reply:
Agreed, perhaps:
```crystal
alias Amber::ParamValue = String | Amber::Router::Files::File | Nil
# then
String => Amber::ParamValue
```

Reply:
This has change since the union here interferes with `Granite::ORM` when it shouldn't